### PR TITLE
Deploy neutron-mlnx-agent and neutron-eswitchd containers

### PR DESCRIPTION
**Pull request by brtknr**
```
git review -X I799993728112a525e34cfbc4e786a10f0ed03be9  # Split out OVS/Linuxbridge agent configs from ml2_conf.ini
git review -X I173669bdf92b1f2ea98907ba16808ca3c914944c  # Deploy neutron-mlnx-agent and neutron-eswitchd containers
git review -X Ifcedcc72307732393a92a702a7567addc043b5b2  # Support neutron_sriov_physnet_mappings to support multiple devices
git review -X I251febe3fd192372177116569f41ccb9aa225302  # Rename eswitchd.conf -> eswitchd.conf.j2
```

### CODE REVIEW

File: ansible/roles/neutron/templates/eswitchd.conf [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[DAEMON]
+fabrics = ibphysnet:{{ neutron_mlnx_interface }}
```
Comment (jovial) [inline ansible/roles/neutron/templates/eswitchd.conf:2]: Should this take a dictionary with mappings from physnet to interface?

Comment (brtknr) [reply ansible/roles/neutron/templates/eswitchd.conf:2]: Yes makes more sense to do this as a mapping.

Comment (brtknr) [reply ansible/roles/neutron/templates/eswitchd.conf:2]: Although i was trying to use the same pattern as physical_interface_mapping in `ansible/roles/neutron/templates/ml2_conf.ini.j2`

**Review by jovial — APPROVED** [review]
Didn't manage to spot anything obviously wrong from a static review, so gets a +1 from me.

**Pull request merged**